### PR TITLE
Configuration is set only for plans (no longer per version and plan)

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ For technical details of KEB, go to the `contributor` directory:
 * [Authorization](./contributor/01-10-authorization.md)
 * [Check API Using Swagger](./contributor/01-20-swagger.md)
 * [Kyma Environment Broker Configuration](./contributor/02-30-keb-configuration.md)
-* [Kyma Environment Broker Configuration for a Given Kyma Version and Plan](./contributor/02-40-broker-configuration-for-given-version-and-plan.md)
+* [Kyma Environment Broker Configuration for a Given Kyma Version and Plan](./contributor/02-40-broker-configuration-for-given-plan.md)
```
Comment (IwonaLanger) [inline docs/README.md:25]:
```suggestion
* [Kyma Environment Broker Configuration for a Given Plan](./contributor/02-40-broker-configuration-for-given-plan.md)
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 2]
```diff
@@ -3,31 +3,24 @@
 > [!NOTE] 
 > This document has been deprecated. Kyma version configuration is not possible.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:4]: You can delete this note now.

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 3]
```diff
@@ -3,31 +3,24 @@
...
 
-Some processes in Kyma Environment Broker (KEB) can be configured to deliver different results. KEB needs a ConfigMap with a configuration for the given Kyma version and plan to process the requests. 
-A default configuration must be defined for the chosen Kyma version. This configuration must be recognized by KEB as applicable for all the supported plans. You can also set a separate configuration for each plan.
+Some processes in Kyma Environment Broker (KEB) can be configured to deliver different results. KEB needs a ConfigMap with a configuration for the given Kyma plan to process the requests.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:6]:
```suggestion
Some Kyma Environment Broker (KEB) processes can be configured to deliver different results. KEB needs a ConfigMap with the configuration for a given Kyma plan to process the requests. 
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 4]
```diff
@@ -3,31 +3,24 @@
 > [!NOTE] 
 > This document has been deprecated. Kyma version configuration is not possible.
 
-Some processes in Kyma Environment Broker (KEB) can be configured to deliver different results. KEB needs a ConfigMap with a configuration for the given Kyma version and plan to process the requests. 
-A default configuration must be defined for the chosen Kyma version. This configuration must be recognized by KEB as applicable for all the supported plans. You can also set a separate configuration for each plan.
+Some processes in Kyma Environment Broker (KEB) can be configured to deliver different results. KEB needs a ConfigMap with a configuration for the given Kyma plan to process the requests. 
+A default configuration must be defined. This configuration must be recognized by KEB as applicable for all the supported plans. You can also set a separate configuration for each plan.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:7]:
```suggestion
The default configuration must be defined. KEB must recognize this configuration as applicable to all supported plans. You can also set a separate configuration for each plan.
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 5]
```diff
@@ -3,31 +3,24 @@
...
   
-While processing requests, KEB reads a configuration from a ConfigMap which holds data about processable Kyma versions and configurable units for a given plan. Only one ConfigMap can exist for a given Kyma version, but it also can be set for multiple Kyma versions if the configuration is the same for every targeted version.
-
+While processing requests, KEB reads a configuration from a ConfigMap which holds data for a given plan.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:9]:
```suggestion
While processing requests, KEB reads the configuration from the ConfigMap that holds data for a given plan.
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 6]
```diff
@@ -3,31 +3,24 @@
...
 > [!NOTE] 
 > Create all configurations in the `kcp-system` namespace.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:11]: Can you explain briefly why?

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 7]
```diff
@@ -3,31 +3,24 @@
...
 
 ```yaml
 keb-config: "true"
-runtime-version-{KYMA_VERSION}: "true"
 ```
 
-You can assign more than one ```runtime-version-{KYMA_VERSION}: "true"``` label as long as the configuration is the same for the provided Kyma versions.
-
 > [!NOTE] 
 > Each ConfigMap that defines a configuration must have both labels assigned.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:25]:
```suggestion
> Each ConfigMap that defines the configuration must have this label assigned.
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 8]
```diff
@@ -60,7 +53,7 @@ You must define a default configuration that is selected when the supported plan
 > [!NOTE] 
 > `kyma-template` and `additional-components` configurations are required.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:54]:
```suggestion
> The `kyma-template` and `additional-components` configurations are required.
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 9]
```diff
@@ -60,7 +53,7 @@ You must define a default configuration that is selected when the supported plan
...
 
-See the example of a ConfigMap with a configuration for Kyma version `2.5.3` and `plan1`, `plan2`, and `trial` plans:
+See the example of a ConfigMap with a default configuration for Kyma and specific configurations for `plan1`, `plan2`, and `trial` plans:
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:56]:
```suggestion
See the example of a ConfigMap with the default configuration for Kyma and specific configurations for `plan1`, `plan2`, and `trial` plans:
```

File: docs/contributor/02-40-broker-configuration-for-given-plan.md [hunk 10]
```diff
@@ -3,31 +3,24 @@
...
+A default configuration must be defined. This configuration must be recognized by KEB as applicable for all the supported plans. You can also set a separate configuration for each plan.
   
-While processing requests, KEB reads a configuration from a ConfigMap which holds data about processable Kyma versions and configurable units for a given plan. Only one ConfigMap can exist for a given Kyma version, but it also can be set for multiple Kyma versions if the configuration is the same for every targeted version.
-
+While processing requests, KEB reads a configuration from a ConfigMap which holds data for a given plan.
 > [!NOTE] 
 > Create all configurations in the `kcp-system` namespace.
 
 > [!NOTE] 
 > Currently, only the Kyma custom resource template and the additional components list can be configured.
```
Comment (IwonaLanger) [inline docs/contributor/02-40-broker-configuration-for-given-plan.md:14]: You have removed the `additional components` from the code, so is the info about them in this note still valid?

File: docs/contributor/02-40-broker-configuration-for-given-plan.md (hunk 6 above)
Comment (piotrmiskiewicz) [reply docs/contributor/02-40-broker-configuration-for-given-plan.md:11]: this note is not necessary, I'll remove it

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**